opengrok-indexer/src/main/java/org/opengrok/indexer/framework/PluginClassLoader.java

```diff
@@ -56,7 +56,8 @@
             "org.opengrok.indexer.authorization.plugins.*",
             "org.opengrok.indexer.authorization.AuthorizationException",
             "org.opengrok.indexer.util.*",
-            "org.opengrok.indexer.logger.*"
+            "org.opengrok.indexer.logger.*",
+            "org.opengrok.indexer.Metrics"
```

Comment:
Could you please explain this to me? I'm probably missing something since I don't know how the plugins in OpenGrok work exactly. I assume that you would like to allow plugins to use the `Metrics` class. However, from my understanding the class is uniquely identified by the name and the classloader that loaded it. Therefore, if the `Metrics` class would be used by the plugin, then it might be a completely different class and the metrics won't be seen/published by the `web` module.

Reply:
Oh, it seems that it is delegating class loading to the tomcat's class loader for some specific cases. Please ignore.